lib/main/atom/tooltipManager.ts

```diff
@@ -15,7 +15,10 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any> el).querySelectorAll(selector)
+  if (!found) {
+    found = (<any> el).rootElement.querySelectorAll(selector)
+  }
```

Comment:
If you're in here, might as well:

1) Use `querySelector` since we only care about the first element
2) Get rid of the `<any>` type cast. How can you ensure that this function always gets a valid HTMLElement?
3) Check that `el.rootElement` is defined first. If on Atom 1.19 you search for a non-existent selector, this will blow up.